### PR TITLE
Add our logo to the docs [revision requested]

### PR DESCRIPTION
**Pull request by pde**
# vanity

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .. This file contains a series of comments that are used to include sections of this README in other files. Do not modify these comments unless you know what you are doing. tag:intro-begin
 
+.. image:: certbot-logo.svg
```
Comment (bmw) [inline README.rst]: When having GitHub render this file, it shows a broken image. This is true both when checking out the branch on the main page of the repo and when clicking "View" in the top right corner of the PR screen. This to me suggests there might be a problem here.

**Comment by vzhz** [comment]
Hello @pde, I'm interested in becoming a first-time contributor and browsing through your issues.   I'm curious about test coverage can be ~99% for a logo being added which seems like it would either be there or wouldn't be.  Also a little confused about why the issue seems to be resolved but isn't closed.  I assume there's some process I'm not familiar with happening and if you have a moment to explain, I would appreciate it.

**Comment by pde** [comment]
@bmw the main aim was to get a Certbot logo on our Github project page, though having it in other places is nice too.  Will investigate getting that to work...

**Comment by bmw** [comment]
Closing this as this PR hasn't seen any progress in nearly 3 months. Reopen when it's ready and I'll happily review again.

**Pull request closed without merging**